kubernetes/provider_test.go

```diff
@@ -167,9 +167,11 @@ func testAccPreCheck(t *testing.T) {
 				"KUBE_CLUSTER_CA_CERT_DATA",
 			}, ", "))
 	}
+}
 
+func skipIfNoGoogleCloudSettingsFound(t *testing.T) {
 	if os.Getenv("GOOGLE_PROJECT") == "" || os.Getenv("GOOGLE_REGION") == "" || os.Getenv("GOOGLE_ZONE") == "" {
-		t.Fatal("GOOGLE_PROJECT, GOOGLE_REGION and GOOGLE_ZONE must be set for acceptance tests")
+		t.Skip("GOOGLE_PROJECT, GOOGLE_REGION nor GOOGLE_ZONE are set, skipping")
```

Comment:
I might not agree with the exact wording, but I agree it should be changed to explain the situation better. I'll change it.